plugins/woocommerce/client/blocks/assets/js/blocks/cart-checkout-shared/payment-methods/express-payment/style.scss

```diff
@@ -25,16 +30,15 @@ $border-width: 1px;
 }
 
 .wc-block-components-express-payment--checkout {
-	/* stylelint-disable-next-line function-calc-no-unspaced-operator */
-	margin-top: calc($universal-border-radius * 3);
+	transition: none !important;
 
 	.wc-block-components-express-payment__event-buttons {
 		list-style: none;
 		display: grid;
-		grid-template-columns: repeat(auto-fit, minmax(240px, 1fr));
+		grid-template-columns: repeat(auto-fit, minmax(150px, 1fr));
 		grid-gap: 12px;
 
-		@include breakpoint("<782px") {
+		@include cart-checkout-mobile-container {
 			grid-template-columns: 1fr;
```

Comment:
Changed this to only force 1 button/row for mobile, we can benefit on tablet from having more buttons on a row

Reply:
Do we need to communicate this change?

Reply:
it will be mentioned in the feature highlight. I don't think it needs some separate comms. WDYT?

Reply:
I would only advise being clear and loud that there are changes to the number of buttons displayed by default.